identity-apps-core/apps/authentication-portal/src/main/webapp/self-registration.jsp

```diff
@@ -234,7 +250,12 @@
 
                 useEffect(() => {
                     if (error && error.code) {
-                        setApiError(error);
+                        const errorDetails = getI18nKeyForError(error.code);
+                        const errorPageURL = authPortalURL + "/registration_error.do?" + "ERROR_MSG="
+                            + errorDetails.message + "&" + "ERROR_DESC=" + errorDetails.description + "&" + "SP_ID="
+                            + "<%= spId %>" + "&" + "REG_PORTAL_URL=" + authPortalURL + "/register.do";
```

Comment:
Shouldn't this be changed as follows?

```suggestion
                            + "<%= spId %>" + "&" + "REG_PORTAL_URL=" + registrationPortalURL + "/register.do";
```

Reply:
This is the base path of the authentication portal. Since this is currently in the authenticationendpoint webapp and not a dedicated registration webapp kept the reference aligned with the authenticationendpoint.